.gitignore

```diff
@@ -0,0 +1,2 @@
+cmd/carwings/carwings
+.idea
```

Comment:
This is fine, but for things like IDEs I prefer to use a global `.gitignore` file so I don't have to do this in every repo.

In my `~/.gitconfig`:

```
[core]
        excludesfile = ~/.gitignore
```

and in my `~/.gitignore`:

```
*~
.#*
*#
.hg
```

(I'm an emacs user.)